index.php

```diff
@@ -1,13 +1,49 @@
 <?php
-require 'functions.php';
+include_once 'functions.php';
+?>
 
-echo '<h1>Record Collection</h1>';
-echo '<br>';
-echo '<br>';
-
-$db = getDB();
-$query = $db->prepare('SELECT `Title`, `Rating`, `Released`, `Condition`, `Artist` FROM `records`;');
-$query->execute();
-$records = $query->fetchALL();
-
-displayRecords($records);
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link href="normalize.css" rel="stylesheet" type="text/css">
+        <link href="style.css" rel="stylesheet" type="text/css">
+        <title>Martin's Record Collection</title>
+    </head>
+    <body>
+    <div class="heading"></div><h1>Martin's Record Collection</h1></div>
+        <?php
+        $db = getDB();
+        echo display(extractFromDB($db));?>
+    <div class="container">
+    <h3>Want to add to your collection? Insert your new record below.</h3><br></head>
```

Comment:
Remove this `</head>` tag at the end of the line.... and then it's good to go! 😁